### PR TITLE
CHANGELOG: add release note for v2.0.10

### PR DESCRIPTION
**Pull request by winkyao**
### What problem does this PR solve? 

Add release note for v2.0.10.

### What is changed and how it works?

---
This change is [Reviewable](https://reviewable.io/reviews/pingcap/tidb/8721)

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,28 @@
 
 All notable changes to this project will be documented in this file. See also [Release Notes](https://github.com/pingcap/docs/blob/master/releases/rn.md), [TiKV changelog](https://github.com/pingcap/tikv/blob/master/CHANGELOG.md) and [PD changelog](https://github.com/pingcap/pd/blob/master/CHANGELOG.md).
 
+## [2.0.10] - 2018-12-18
+- Fix the possible issue caused by canceling a DDL job [#8513](https://github.com/pingcap/tidb/pull/8513)
+- Fix the issue that the `ORDER BY` and `UNION` statements cannot quote the column including a table name [#8514](https://github.com/pingcap/tidb/pull/8514)
```
Comment (jackysp) [inline CHANGELOG.md]: statements -> clauses?

File: CHANGELOG.md [hunk 2]
```diff
@@ -2,6 +2,28 @@
...
+- Fix the issue that the `UNCOMPRESS` function does not judge the incorrect input length [#8607](https://github.com/pingcap/tidb/pull/8607)
+- Fix the issue encountered by `ANSI_QUOTES SQL_MODE` when upgrading TiDB [#8575](https://github.com/pingcap/tidb/pull/8575)
+- Fix the issue that `select` returns the wrong result in some cases [#8570](https://github.com/pingcap/tidb/pull/8570)
+- Fix the possible issue that TiDB cannot exit when it receives the exit signal [#8501](https://github.com/pingcap/tidb/pull/8501)
+- Fix the issue that `IndexLookUpJoin` returns the wrong result in some cases [#8508](https://github.com/pingcap/tidb/pull/8508)
+- Avoid pushing down the filter containing `GetVar` or `SetVar` [#8454](https://github.com/pingcap/tidb/pull/8454)
+- Fix the issue that the result length of the `UNION` statement is incorrect in some cases [#8491](https://github.com/pingcap/tidb/pull/8491)
```
Comment (jackysp) [inline CHANGELOG.md]: ditto

**Comment by winkyao** [comment]
@jackysp PTAL

File: CHANGELOG.md [hunk 3]
```diff
@@ -2,6 +2,28 @@
 
 All notable changes to this project will be documented in this file. See also [Release Notes](https://github.com/pingcap/docs/blob/master/releases/rn.md), [TiKV changelog](https://github.com/pingcap/tikv/blob/master/CHANGELOG.md) and [PD changelog](https://github.com/pingcap/pd/blob/master/CHANGELOG.md).
 
+## [2.0.10] - 2018-12-18
+- Fix the possible issue caused by canceling a DDL job [#8513](https://github.com/pingcap/tidb/pull/8513)
+- Fix the issue that the `ORDER BY` and `UNION` statements cannot quote the column including a table name [#8514](https://github.com/pingcap/tidb/pull/8514)
+- Fix the issue that the `UNCOMPRESS` function does not judge the incorrect input length [#8607](https://github.com/pingcap/tidb/pull/8607)
+- Fix the issue encountered by `ANSI_QUOTES SQL_MODE` when upgrading TiDB [#8575](https://github.com/pingcap/tidb/pull/8575)
+- Fix the issue that `select` returns the wrong result in some cases [#8570](https://github.com/pingcap/tidb/pull/8570)
```
Comment (zz-jason) [inline CHANGELOG.md]:
```suggestion
 - Fix the issue that `SELECT` returns the wrong result in some cases [#8570](https://github.com/pingcap/tidb/pull/8570)
```

File: CHANGELOG.md [hunk 4]
```diff
@@ -2,6 +2,28 @@
...
+- Fix the possible issue that TiDB cannot exit when it receives the exit signal [#8501](https://github.com/pingcap/tidb/pull/8501)
+- Fix the issue that `IndexLookUpJoin` returns the wrong result in some cases [#8508](https://github.com/pingcap/tidb/pull/8508)
+- Avoid pushing down the filter containing `GetVar` or `SetVar` [#8454](https://github.com/pingcap/tidb/pull/8454)
+- Fix the issue that the result length of the `UNION` clauses is incorrect in some cases [#8491](https://github.com/pingcap/tidb/pull/8491)
+- Fix the issue of `PREPARE FROM @var_name` [#8488](https://github.com/pingcap/tidb/pull/8488)
+- Fix the panic issue when dumping statistics information in some cases [#8464](https://github.com/pingcap/tidb/pull/8464)
+- Fix the statistics estimation issue of point queries in some cases [#8493](https://github.com/pingcap/tidb/pull/8493)
+- Fix the panic issue when the returned default `enum` value is a string #[8476](https://github.com/pingcap/tidb/pull/8476)
+- Fix the issue that too much memory is consumed in the scenario of wide tables [#8458](https://github.com/pingcap/tidb/pull/8458)
```
Comment (zz-jason) [inline CHANGELOG.md]: I think the issue link is wrong, we should post the 2.0 PR link, not the master's

**Comment by winkyao** [comment]
@jackysp @zz-jason PTAL

**Pull request merged**